app/views/partials/header.ejs

```diff
@@ -1,10 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-
     <meta charset="utf-8">
     <meta data-build-hash="<%=commitHash%>">
     <meta name="viewport" content="width=device-width">
+    <% if((typeof article !== 'undefined') || (typeof user !== 'undefined') || title === "Pinch of Code") { %>
```

Comment:
Do google index only pages when it's logged in?

Reply:
That's not what this is doing, as meta tags must be defined in head, the current setup is, if there's an article that means it's on an article route, if user is there it's on an author route, or if it's that title it's on the homepage.

I do not believe any other pages, until we create about us, should be indexed. As defined in robots.txt